Task-based edit: User Subscriptions (#6134)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/managing_user_subscriptions.md

### Updating email subscription states

There are four ways to update a user's email subscription state:

#### SDK integration

Use the Braze SDK to update a user's subscription state.

#### REST API

Use the [`/users/track` endpoint][users-track] to update the [`email_subscribe`][user_attributes_object] attribute for a given user.

#### User profile

To manually change the subscription status, do the following:

1. On the **[Search Users][5]** page, find the user profile.
2. Select the **Engagement** tab and click the **Unsubscribed**, **Subscribed**, or **Opted In** buttons to change that user's subscription status.

If available, the user profile also displays a timestamp for when the user's subscription was last changed.

#### Preference center

> [Preference center](#email-preference-center) Liquid can be included at the bottom of your emails, allowing users to opt-in or opt-out of emails. Braze manages the subscription state updates from the preference center.

Reviewer: ```suggestion
[Preference center](#email-preference-center) Liquid can be included at the bottom of your emails, allowing users to opt in or out of emails. Braze manages the subscription state updates from the preference center.
```

### Checking email subscription state

![User profile for John Doe with their email subscription state set to Subscribed.][3]{: style="float:right;max-width:35%;margin-left:15px;"}

There are two ways you can check a user's email subscription state with Braze:

> #### User Profile

Reviewer: ```suggestion
#### User Profile
```
I think the previous numbered list format works in this case with the preceding sentence. Maybe we can edit the text to match what you suggested. Something like:

1. **User profile**: Find the user's profile on the **[Search Users][5]** page, then select the **Engagement** tab to view and manually update a user's subscription state.
2. **REST API export**: Use the [Export users by segment][segment] or [Export users by identifier][identifier] endpoints to export individual user profiles in JSON format.

🤔

Author: Ooh, that's really good. Thank you!

Reviewer: I think you might've made this change above instead of replacing these headers? 🤔

Author: My gosh, I hate when there are multiple headers with the same names. Thank you~

> Use the [Subscription Group REST APIs][25] to programmatically manage the subscription groups that you have stored on the Braze dashboard to the **Subscription Group** page.

Reviewer: ```suggestion
Use the [Subscription Group endpoints][25] to programmatically manage the subscription groups that you have stored on the Braze dashboard to the **Subscription Group** page.
```
Can you also update the link for [25]? Seems to be an old redirect that links to what's now https://www.braze.com/docs/api/endpoints/subscription_groups

![Email with Set Custom Plaintext Footer option selected.][23]{: style="max-width:70%" }

### Creating a custom unsubscribe page

When users click an unsubscribe URL in an email, they are taken to a default landing page that confirms the change to their subscription.

To create a custom landing page that users will be directed to (instead of the default page) upon subscribing, go to **Email Preferences** > **Subscription Pages and Footers** and provide the HTML for your custom landing page. We recommend including a resubscribe link (i.e., `{{${set_user_to_subscribed_url}}}` ) on the landing page so that users have the option to resubscribe in case they unsubscribed by accident.

![Custom unsubscribe email in the Custom Unsubscribe Page panel.][11]

{% endraw %}

### Creating a custom opt-in page

Instead of immediately subscribing a user to your email campaigns, creating a custom opt-in page can give your users the opportunity to acknowledge and control their notification preferences. This additional communication can also help your email campaigns stay out of the spam folder since your users will have chosen to be opted-in. 

> Go to **Email Preferences > Subscription Pages and Footers**, and customize the styling in the **Custom Opt-In Page** section to see how that indicates to your users that they've been subscribed.

Reviewer: ```suggestion
Go to **Email Preferences** > **Subscription Pages and Footers**, and customize the styling in the **Custom Opt-In Page** section to see how that indicates to your users that they've been subscribed.
```
